src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -347,9 +349,11 @@ private Claims(
 
                 private boolean containsClaims(
                     String issuerName,
-                    String audienceName)
+                    String[] audienceNames)
                 {
-                    return Objects.equals(this.issuerName, issuerName) && Objects.equals(this.audienceName, audienceName);
+                    return (this.issuerName == null || Objects.equals(this.issuerName, issuerName))
+                            && (this.audienceName == null
+                            || (audienceNames != null && Arrays.asList(audienceNames).contains(this.audienceName)));
```

Comment:
In each case, move `&&`, `||` operators to end of previous line, rather than start of next line.

Avoid implicit allocation `Arrays.asList(...)` during logically readonly call to `containsClaims`. Suggest implementing `indexOf` helper method instead.